### PR TITLE
feat(guide): add Angular Bootstrap Boilerplate link in Angular 2.x+ resources

### LINKED ISSUE #35595: Please add Angular-Bootstrap-Boilerplate
Please consider adding the [Angular Bootstrap Boilerplate](https://github.com/mdbootstrap/Angular-Bootstrap-Boilerplate) to https://guide.freecodecamp.org/angular/angular-resources/

It's a useful, open-sourced Angular CRUD application starter with NgRx state management and Firebase backend.

### PR DESCRIPTION
**Pull request by guttume**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #35595

### CODE REVIEW

File: guide/english/angular/angular-resources/index.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ A collection of helpful Angular resources
 *   <a href='https://angular.io/' target='_blank' rel='nofollow'>Angular</a> - The Angular Homepage
 *   <a href='https://angular.io/guide/styleguide' target='_blank' rel='nofollow'>Angular Style Guide</a> - Detailed best practices for Angular Development
 *   <a href='https://material.angular.io/' target='_blank' rel='nofollow'>Angular Material</a> - Excellent UI Framework that was built for Angular
+* <a href='https://github.com/mdbootstrap/Angular-Bootstrap-Boilerplate' target='_blank' rel='nofollow'>Angular Bootstrap Boilerplate</a> - Free, responsive CRUD application starter with NgRx state management, Firebase backend and material design
```
Comment (thecodingaviator) [inline guide/english/angular/angular-resources/index.md]:
```suggestion
*   <a href='https://github.com/mdbootstrap/Angular-Bootstrap-Boilerplate' target='_blank' rel='nofollow'>Angular Bootstrap Boilerplate</a> - Free, responsive CRUD application starter with NgRx state management, Firebase backend and material design
```

Comment (RandellDawson) [reply guide/english/angular/angular-resources/index.md]: @thecodingaviator Not sure I understand why this change is needed.  Does it change how it displays when viewing it locally?

Comment (thecodingaviator) [reply guide/english/angular/angular-resources/index.md]: Nope, it is to make it consistent with the rest of the file. Either way is fine and displays the same

**Comment by thecodingaviator** [comment]
@guttume Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Comment by guttume** [comment]
@thecodingaviator Thank you so much. This was my first contribution ever. Thank you for the opportunity.

**Comment by thecodingaviator** [comment]
@guttume Thanks for contributing to the freeCodeCamp repository. Congratulations on your first PR! 🎊 
Look forward to more contributions from you! ✈️

**Pull request merged**